src/main/kotlin/three/consulting/epoc/service/ProjectService.kt

```diff
@@ -61,6 +61,12 @@ class ProjectService(private val projectRepository: ProjectRepository) {
             throw UnableToDeleteProjectException(projectId)
         }
     }
+
+    fun findAllProjects(): MutableList<Project> {
+        var projectList = projectRepository.findAll()
+        projectList.forEach { ProjectDTO(it) }
+        return projectList
+    }
```

Comment:
Let's not use MutableLists at all. Kotlin scoped functions usually provide a way to return a manipulated object instead of first initiating a mutable one. Also using the list, set, whatever in the variable name is overly complex. So how about
```kotlin
val projects = projectRepository.findAll()
return projects.map { ProjectDTO(it) }
```